catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/SeedCommandTest.java

```diff
@@ -107,7 +107,7 @@ public void testSources() throws Exception {
 
   @Test
   public void testCql() throws Exception {
-    final String cql = "modified AFTER 2016-07-21T00:00:00Z";
+    final String cql = "modified AFTER 2016-07-21T00:00:00+00:00";
```

Comment:
It errors out if I use the Z version, so maybe?

Reply:
it may be system dependent. I have had issues with the old java time api's and toString, if they aren't using a date formatter it could change depending on the system.

Reply:
I believe it can still read in the zulu formatted time but it only writes out the time in UTC+offset